utils/test/Is_LSMB_running.sh

```diff
@@ -13,6 +13,20 @@ if [[ -z $EX_NOHOST ]]; then
     exit 99;
 fi
 
+
+# base url for lsmb defaults to http://localhost:5001
+# this can be overriden by setting environment variable $LSMB_BASE_URL
+lsmb_base_url=${LSMB_BASE_URL:-http://localhost:5001}
+
+# We check Starman is running on a specific port
+url_without_port=${lsmb_base_url%:[[:digit:]]*}
+lsmb_port=${lsmb_base_url#$url_without_port}
+lsmb_port=${lsmb_port#:}  # strip colon
+
+# Default to port 5001
+lsmb_port=${lsmb_port:-5001}
```

Comment:
We should also handle the Travis port variable here. Unless thats been replaced replacer since I last checked